### ISSUE TITLE
Add Functionality to Quiz Skip Button

### ISSUE THREAD

**Issue by jovecalimlim** [info442-aut21/f-team#37]
When clicked, the Main Page is loaded with no filters applied to the Main Page.

This issue is part of completing requirement 1.1.

Depends on #36

**Comment by jovecalimlim** [comment]
Skip button not being added, this issue is no longer needed
